src/Validator/Rules/ValuesOfCorrectType.php

```diff
@@ -56,12 +65,17 @@ static function unknownFieldMessage($typeName, $fieldName, $message = null)
     public function getVisitor(ValidationContext $context)
     {
         return [
```

Comment:
You could put `$fieldName` at the top of the visitor function and then `use` it inside closures instead of having it defined as static.

Reply:
Resolved in 9dc41f0